packages/react-notifications/src/components/notification-kinds/api-error-message/api-error-message.tsx

```diff
@@ -62,7 +73,14 @@ const FormattedErrorMessage = (props: Props) => {
 
   if (messageCode) {
     // The `error` object might contain extra fields for the specific `code`.
-    return <>{intl.formatMessage(messageCode, props.error)}</>;
```

Comment:
The changes here are related to the fact that `formatMessage` expects a map of scalar values. If we just pass `props.error`, it can contain nested objects and so the compiler rightfully complains.